shared/js/background/broken-site-report.js

```diff
@@ -321,6 +321,8 @@ export async function breakageReportForTab({
 export async function sendBreakageReportForCurrentTab({ pixelName, currentTab, category, description, reportFlow }) {
     await settings.ready();
     await tdsStorage.ready('config');
```

Comment:
Any reason to wait for both `tdsStorage.ready('config')` and `tdsStorage.config.allLoadingFinished`?